src/main/java/com/gachigang/ontherun/config/security/SecurityBeans.java

```diff
@@ -43,4 +51,14 @@ public AuthenticationManager authenticationManager (AuthenticationConfiguration
     PasswordEncoder bCryptPasswordEncoder() {
         return new BCryptPasswordEncoder();
     }
+
+    @Bean
+    CorsConfigurationSource corsConfigurationSource() {
+        CorsConfiguration configuration = new CorsConfiguration();
+        configuration.setAllowedOrigins(Arrays.asList(ALLOWEDORIGINS));
+        configuration.setAllowedMethods(Arrays.asList(ALLOWEDMETHODS));
+        UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
+        source.registerCorsConfiguration("/**", configuration);
```

Comment:
I believe we want to allow some headers (A lot of hackers use headers in their attact), so we don`t want to allow it. I suggest for now allow only "Authorization", "Cache-Control" and "Content-Type" headers, it can be hardcoded -- because we will never change it for different instances

Reply:
I`ve added "setAllowedHeaders"